src/v0/destinations/facebook_pixel/utils.js

```diff
@@ -192,10 +184,6 @@ const transformedPayloadData = (
         delete clonedCustomData[eventProp];
       }
     }
-    const isCustomProperty = customEventProperties[eventProp] || false;
-    if (isStandard && !isCustomProperty && !isDefaultPiiProperty) {
-      delete clonedCustomData[eventProp];
-    }
```

Comment:
Could you please let me know why you're removing this entirely? It does not seem the same with device mode implementation.